### ISSUE TITLE
Is there a Support Forum?

### ISSUE THREAD

**Issue by zxxz** [griffon/griffon#70]
Hello,
Is there a forum on griffon? I haven't been able to find any (working) link to a forum or mailing list.

If it wasn't for the 2 releases in 2015 I would have thought this project is dead...

Am I missing something?

Thanks,
zxxz

**Comment by aalmiray** [comment]
We had one at Nabble but it appears to be offline. I'll take care of a more permanent solution.

**Comment by zxxz** [comment]
@aalmiray Thanks for your answer. Where will you announce the new Forum? 
Most of the support or mailing list links I have found point to codehaus and their services have been terminated.
I am evaluating whether to use griffon for a project and without (community) support I am afraid I will have to dismiss it.
The official tutorial seems to have issues as I posted with a [question on SO](http://stackoverflow.com/questions/30915683/griffon-official-tutorial-unable-to-successfully-compile-griffon-swing-groovy-p).
